flow-typed/box-ui-elements.js

```diff
@@ -658,31 +670,18 @@ type FileAccessStats = {
     preview_count?: number,
 };
 
-type TaskAssignment = {
-    assigned_to: User,
-    id: string,
-    status: TaskAssignmentStatus,
-    type: 'task_assignment',
-};
-
-type TaskAssignments = {
-    entries: Array<TaskAssignment>,
-    total_count: number,
-};
-
+// this is a subset of TaskNew, which imports as `any`
 type Task = {
     created_at: string,
     created_by: User,
-    due_at?: string,
     id: string,
-    message: string,
-    task_assignment_collection: TaskAssignments,
+    permissions: BoxTaskPermission,
     type: 'task',
 };
 
 type Tasks = {
     entries: Array<Task>,
-    total_count: number,
+    next_marker: 'string' | null,
```

Comment:
Are the quotes around "string" intentional?

Reply:
copied from "Collaborators" object but I doubt it should be a literal, fixing